### DISCUSSION TITLE
Disconnect styleManager from the selected component

### DISCUSSION THREAD

**Discussion by hurnell** [GrapesJS/grapesjs#5417] [Q&A]
I'm wondering if its possible to disconnect styleManager from the selected component in the canvas. 

What I'd like to be able to do is load css styles into the styleManager programatically then "catch" and save any style changes without updating the selected component.

**Comment by artf** [comment]
Yes, you can change the style manager target via [`styleManager.select`](https://grapesjs.com/docs/api/style_manager.html#select)
